BankSDK/GiniBankSDK/Tests/GiniBankSDKTests/Skonto/PriceTextFieldTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  PriceTextFieldTests.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniBankSDK
+
+class PriceTextFieldTests: XCTestCase, PriceTextFieldDelegate {
+
+    var priceTextField: PriceTextField!
+    var priceTextChangeExpectation: XCTestExpectation?
+
+    override func setUp() {
+        super.setUp()
+        priceTextField = PriceTextField(frame: .zero)
+        priceTextField.priceDelegate = self
+        priceTextField.text = formatValue(0)
+    }
+
+    override func tearDown() {
+        priceTextField = nil
+        priceTextChangeExpectation = nil
+        super.tearDown()
```

Comment:
[clarification] Should we add also `priceTextField.priceDelegate = nil` inside tearDown method?

Reply:
We can skip it, because delegate is weak inside textField